com.studio08.xbplay.yml

```diff
@@ -0,0 +1,63 @@
+app-id: com.studio08.xbplay
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
I guess you don't need GL?

Reply:
I don't think so? Its working fine without that. I simply copied what the example electron app had (as this is an electron app)